src/ActionSheet/index.tsx

```diff
@@ -53,7 +54,8 @@ export default class ActionSheet extends React.Component<Props, State> {
     (this._actionSheetHeight = nativeEvent.layout.height);
 
   render() {
-    const { isVisible, overlayOpacity } = this.state;
+    const { isVisible, overlayOpacity, options } = this.state;
+    const useModal = options && options.useModal;
```

Comment:
May be worth coercing useModal to a boolean at this point instead of letting it be possibly undefined throughout the render function - makes the intended behavior more obvious if this prop ever becomes on-by-default.

Reply:
Would `const useModal = options && !!options.useModal;` be enough?

Reply:
```suggestion
    const useModal = options ? options.useModal === true : false;
```

Reply:
This guarantees the value will always either be true or false.

Reply:
it looks like options is nullable in the state, so it'd need to be `!!(options && options.useModal)` to be safe because `null && false` is null.

if you add an explicit type annotation as in `const useModal: boolean = ...`, the compiler can help make sure we're not accidentally letting null/undefined creep in to the variable.